stats.lua

```diff
@@ -392,6 +402,54 @@ local function append_display_sync(s)
 end
 
 
+local function append_filters(s, prop, prefix)
+    local length = 0
+    local filters = {}
+
+    local speed = mp.get_property_osd("speed")
+    if speed ~= "1.00" then
+        length = length + string.len("speed []") + speed:len()
+        filters[#filters+1] = "speed" .. it(" [" .. speed .. "]")
+    end
+
+    if prop == "vf" and mp.get_property_bool("deinterlace") then
```

Comment:
I am aware that it has 3 states and also that it's not a filter directly. The doc says (and I tested it) that the deinterlace property will never return the 3rd state (auto), so it should always reflect on whether deinterlacing is active or not. As to which deinterlacing filter is actually inserted, there does not seem to be an API to query that, see my comment on the speed for that. I think that listing deinterlace is better than nothing, though.